prometheusbp/metrics_buckets_spec.go

```diff
@@ -0,0 +1,28 @@
+//go:build metrics_spec
+// +build metrics_spec
+
+package prometheusbp
+
+// DefaultBuckets is the default bucket values for a prometheus histogram metric.
```

Comment:
(nit) Use paragraph breaks

```suggestion
// DefaultBuckets is the default bucket values for a prometheus histogram metric.
//
```

Reply:
Not sure if there's a preferred style between this (with more breaks) and Konrad's comment https://github.com/reddit/baseplate.go/pull/567#discussion_r976845341

Reply:
I think you can do both :slightly_smiling_face: I might have squashed my suggestion too much.